File: 프로그래머스/0/181934. 조건 문자열/조건 문자열.js
const evaluateComparison = (ineq, eq, n, m) => {
    const operator = ineq + eq;
    
    switch(operator){
        case ">=": return n >= m;
        case "<=": return n <= m;
        case ">!": return n > m;
        case "<!": return n < m;
        default: return false;
    }
}

function solution(ineq, eq, n, m) {
    return evaluateComparison(ineq, eq, n, m) ? 1 : 0;
}

File: 프로그래머스/0/181934. 조건 문자열/조건 문자열.py
def solution(ineq, eq, n, m):
    op = ineq + eq
    if op == ">=":
        return 1 if n >= m else 0
    if op == "<=":
        return 1 if n <= m else 0
    if op == ">!":
        return 1 if n > m else 0
    if op == "<!":
        return 1 if n < m else 0